cmd/deploy.go

```diff
@@ -33,13 +33,9 @@ import (
 // deployCmd represents the deploy command
 var deployCmd = &cobra.Command{
 	Use:   "deploy",
-	Short: "A brief description of your command",
-	Long: `A longer description that spans multiple lines and likely contains examples
-and usage of using your command. For example:
-
-Cobra is a CLI library for Go that empowers applications.
-This application is a tool to generate the needed files
-to quickly create a Cobra application.`,
+	Short: "Deploy a new version of your application to your VPS using Sidekick",
+	Long: `This command deploys a new version o your application to your VPS. 
```

Comment:
Small typo here `of your application`

Reply:
Just fixed it. 
Nice and helpful project. I'd happy to help

Reply:
Thank you! 
Happy to hear that! There will be big features we would like to build in the next couple of months. We can discuss more